index.html

```diff
@@ -141,7 +141,9 @@ <h3 class="program-title">Main Program</h3>
             </div>
           </div>
         </div>
-        <button class="join-btn">Join Genuine Charity</button>
+        <div class="join-btn">
+          <a href="#">Join Genuine Charity</a>
+        </div>
```

Comment:
- [ ] Great job so far 👍 Kindly style this element as shown in the required design. In the desktop version, you should see an underlined text. 
- Your design:
![Annotation 2022-04-16 080255](https://user-images.githubusercontent.com/85113567/163665540-25593122-e7a9-4c2f-a190-e4522590eb18.jpg)
- Required design:
![Annotation 2022-04-16 080314](https://user-images.githubusercontent.com/85113567/163665551-ee0f7072-4d7c-461a-a5e2-cb48c21c621e.jpg)

Reply:
Okay cool! :cool: